app/lib/actions/katello/content_view/presenters/incremental_updates_presenter.rb

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2014 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+module Actions
+  module Katello
+    module ContentView
+      module Presenters
+        class IncrementalUpdatesPresenter < Helpers::Presenter::Base
+          HUMANIZED_TYPES = {
+            ::Katello::Erratum::CONTENT_TYPE => "Errata",
+            ::Katello::Package::CONTENT_TYPE => "Packages",
+            ::Katello::PuppetModule::CONTENT_TYPE => "Puppet Modules"
+          }
+
+          def humanized_output
+            if action.output[:changed_content]
+              humanized_content
+            else
+              _("Incremental Update incomplete.")
+            end
+          end
+
+          def humanized_content
+            humanized_lines = []
+
+            action.output[:changed_content].each do |output|
+              cvv = ::Katello::ContentViewVersion.where(:id => output[:content_view_version][:id]).first
```

Comment:
So `find_by_*` is replaced by `find_by` in Rails 4:
```ruby
cvv = ::Katello::ContentViewVersion.find_by_id(id)
cvv = ::Katello::ContentViewVersion.find_by(:id => id)
```
We'll probably do a search and replace. I'd probably prefer that but I'll leave it up to you though.